prow/plugins/releasenote/releasenote_test.go

```diff
@@ -69,36 +71,36 @@ func TestReleaseNoteComment(t *testing.T) {
 			action:        "created",
 			body:          "/release-note",
 			currentLabels: []string{releaseNoteLabelNeeded, "other"},
-
-			shouldComment: true,
```

Comment:
I removed that because the comment is not from an author or member. I did not think it would actually create a comment in that case, but on confirmation it does. I will add it back.